### PR TITLE
support non-x86 arches in gstreamer helper

### PR DESCRIPTION
**Pull request by sharkcz**
fix issue #326

### CODE REVIEW

**Comment by sharkcz** [comment]
Now it tries "gstreamer1(decoder-video/x-h264)()(64bit)" on my ppc64le.

File: contrib/gstreamer-plugin/pk-gstreamer-install.c [hunk 1]
```diff
@@ -232,7 +232,13 @@ pk_gst_get_arch_suffix (void)
 		goto out;
 	}
 
-	g_warning ("PackageKit: did not recognise machine type: '%s'", buf.machine);
+	/* fall back */
+	if (sizeof(void *) == 8) {
+		suffix = "()(64bit)";
+		g_debug("PackageKit: assume 64-bit suffix for machine type: '%s'", buf.machine);
```
Comment (hughsie) [inline contrib/gstreamer-plugin/pk-gstreamer-install.c]: space needed before the bracket please

File: contrib/gstreamer-plugin/pk-gstreamer-install.c [hunk 2]
```diff
@@ -232,7 +232,13 @@ pk_gst_get_arch_suffix (void)
...
+	} else
```
Comment (hughsie) [inline contrib/gstreamer-plugin/pk-gstreamer-install.c]: `} else {` please

File: contrib/gstreamer-plugin/pk-gstreamer-install.c [hunk 3]
```diff
@@ -232,7 +232,13 @@ pk_gst_get_arch_suffix (void)
...
+		g_debug("PackageKit: assume no suffix for machine type: '%s'", buf.machine);
```
Comment (hughsie) [inline contrib/gstreamer-plugin/pk-gstreamer-install.c]: space needed

**Comment by sharkcz** [comment]
Hopefully I got the coding style right this time, thanks for review.

**Pull request merged**